### DISCUSSION TITLE
How to make qpsolvers detect MOSEK

### DISCUSSION THREAD

**Discussion by YuanzhaoLin1999** [qpsolvers/qpsolvers#282] [Q&A] [answered]
I'd like to test different qp algorithms. And I have worked using MOSEK for a long time. I'd like to use it as a benchmark. However, it seems like the qpsolvers package cannot detect my MOSEK, I cannot select 'mosek' as a solver.

**Comment by stephane-caron** [comment, marked as answer]
In a Python interpreter, can you check that the following two imports work?

```python
import cvxopt
import mosek
```

If any of those don't work you can `pip install` the corresponding packages:

```console
$ pip install cvxopt mosek
```

**Reply by YuanzhaoLin1999** [reply]
Yes, it works. I can import mosek buy maybe I don't have cvxopt before so it seems like qpsolver detect MOSEK by cvxopt? 
Thanks a lot !
